rust/examples/mobile-client.rs

```diff
@@ -119,10 +119,10 @@ fn main() -> Result<(), ()> {
 // perform the participant task (this function should be triggered regularly on the phone while the
 // app is active or in a background task)
 fn perform_task(url: &str, bytes: &[u8], model: Model) -> Vec<u8> {
-    let mut client = MobileClient::deserialize(url, bytes);
+    let mut client = MobileClient::deserialize(url, bytes).unwrap();
     client.set_local_model(model);
-    client = client.perform_task();
-    println!("global model: {:?}", client.get_global_model());
+    client = client.perform_task().unwrap();
+    println!("global model: {:?}", client.get_global_model().unwrap());
```

Comment:
Unrelated to this PR but afaik most models will be at list around 1MB large, we may not want to print them.